README.md

```diff
@@ -54,6 +54,7 @@ container images and config changes are propagated to the cluster.
 |[Canva](https://www.canva.com/)|[Chattigo](https://chattigo.com)|[Cloudlets](https://cloudlets.io)|
 |[Cloudogu GmbH](https://cloudogu.com)|[CLVR Cloud Ltd](https://clvr.cloud)|[College of William & Mary](https://www.wm.edu)|
 |[Control Plane](https://control-plane.io)|[Consorcio Chile](https://www.consorcio.cl)|[CrowdRiff](https://crowdriff.com)|
+|[Duedil](https://www.duedil.com/)
```

Comment:
Do you need a `|` at the end of this line?

Reply:
It needs to be merged in after `|[Cybrary](https://cybrary.it)|`, which means all other entries will move one position (either to the right, or one line lower).

Reply:
We need a better formatting options for this. really is not clear to follow and format this.
i can see this is superseded by #3234